etna/packages/etna-js/upload/workers/uploader.jsx

```diff
@@ -174,14 +176,18 @@ export default (uploader) => {
 
     remove: (upload) => uploader.dispatch({ type: 'REMOVE_UPLOAD', upload }),
 
-    timeout: () => {
-      ++uploader.timeouts;
-      if (uploader.timeouts == uploader.maxTimeouts) {
-        uploader.reset();
```

Comment:
Removing the call to `reset()` seems like it might change the behaviour of the uploader -- have you tested re-uploading the same file after it times out?

Reply:
Good call, I locally am resetting at this point.  Haven't pushed.